src/gui/plugins/spawn/Spawn.cc

```diff
@@ -190,7 +190,7 @@ void Spawn::LoadConfig(const tinyxml2::XMLElement *)
 }
 
 
-// TODO(ahcorde): Replace this when this function is on ign-rendering6
+// TODO(ahcorde): Replace this when this function is on ign-rendering7
```

Comment:
Unintentional change. But in any case this comment is going away once https://github.com/ignitionrobotics/ign-gazebo/pull/1432/ is forward-ported.

Reply:
https://github.com/ignitionrobotics/ign-gazebo/pull/1444/commits/2117f1c4ba31917ffd83064d14d1ac0ccc765dc2